.circleci/soltest_all.sh

```diff
@@ -69,7 +69,11 @@ for OPTIMIZE in ${OPTIMIZE_VALUES[@]}
 do
     for EVM in ${EVM_VALUES[@]}
     do
-        [[ " $RUN_STEPS " =~ " $STEP " ]] && EVM="$EVM" OPTIMIZE="$OPTIMIZE" BOOST_TEST_ARGS="-t !@nooptions" "${REPODIR}/.circleci/soltest.sh"
+        # run tests against hera ewasm evmc vm, only if OPTIMIZE == 0 and evm version is byzantium
+        EWASM_ARGS=""
+        [ "${EVM}" = "byzantium" ] && [ "${OPTIMIZE}" = "0" ] && EWASM_ARGS="--ewasm"
+
+        [[ " $RUN_STEPS " =~ " $STEP " ]] && EVM="$EVM" OPTIMIZE="$OPTIMIZE" SOLTEST_FLAGS="$EWASM_ARGS" BOOST_TEST_ARGS="-t !@nooptions" "${REPODIR}/.circleci/soltest.sh"
```

Comment:
This disables any SOLTEST_FLAGS passed in from outside, right? Shouldn't we change it to `SOLTEST_FLAGS="$SOLTEST_FLAGS $EWASM_ARGS"`?

Reply:
Thats true